src/StoreKeeper/WooCommerce/B2C/Models/TaskModel.php

```diff
@@ -200,7 +207,7 @@ public static function countTasksByCreatedDateTimeRange($start, $end): int
     {
         global $wpdb;
 
-        $select = static::prepareCreatedDateTimeRangeSelect($start, $end);
+        $select = static::prepareProcessedDateTimeRangeSelect($start, $end);
```

Comment:
the `public static function countTasksByCreatedDateTimeRange($start, $end): in` is not correct anymore

Reply:
this function was also used in counting of the incomming tasks

Reply:
Reverted to created date.